home/ivp-planning/getting-started/importing-pdfs.md

```diff
@@ -2,30 +2,28 @@
 
 In iVP it is possible to import PDFs that will act as an overlay for the [floor plan](../user-interface/the-floor-plan.md). This can be useful for illustration or when it is necessary to transform a 2D (factory) plan into a 3D scene.
 
+## Importing PDFs step by step:
 
-{% embed url="https://youtu.be/MuLt94b64O8?t=203" %}
-How to import and export PDFs (german)
-{% endembed %}
+To import a PDF, click on **File -> Import**. A new window will open that allows you to navigate to the folder containing the PDF you want to import.
 
-## Importing PDFs step by step:
+![](../../../.gitbook/assets/pdf_import_pdf_menu_entry.jpg)
+
+As soon as you have found the file, simply double-click on it or select it and click **Open**.
+
+![](../../../.gitbook/assets/pdf_import_pdf_file_dialogue.jpg)
 
-To import a PDF click on **File -> PDF -> Import PDF**. A new window will open that allows you to navigate to the folder that contains the PDF you want to import.
+Before iVP actually processes the PDF and imports it, you will be asked to set some parameters that specify how it will be loaded. The options are:
 
-![](../../../.gitbook/assets/iVP\_pdf\_import\_pfd\_menu\_entry.jpg)
+* **Page Size**: Changes the size of the imported PDF, you can either choose from a preset or manually customize the size of the file.
 
-As soon as you found the file, simply double click or it or select it and click **Open**.
+* **Width Original File**: Displays the width of the original file.
```

Comment:
Can also be changed when the page size is not taken from the file. Also explain what this option actually does, when modified.